config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
 
   resource :notifications, only: [:edit, :update]
+  resource :dwp_warnings
```

Comment:
Can we limit this to only edit and update?

Reply:
yes, I forget about that one :)